lfs/lfs.go

```diff
@@ -16,10 +16,6 @@ import (
 	"github.com/rubyist/tracerx"
 )
 
-const (
-	Version = "1.5.0"
```

Comment:
Ah, that's because the `github.com/git-lfs/git-lfs/lfs` package already has a dependency pointing towards `github.com/git-lfs/git-lfs/config`, introducing another dependency the other way around would be a circular dependency issue. Let's just leave it in the `config` package for now, even though it's kind of unfortunate.

Untangling that stuff would be more time than I think it's worth at this stage.